src/main/java/lotto/LottoTVShow.java

```diff
@@ -1,32 +1,55 @@
 package lotto;
 
 import lotto.domain.Lotto;
+import lotto.domain.LottoMatchResult;
 import lotto.domain.LottoResult;
+import lotto.model.LottoPrizePolicy;
+import lotto.view.InputView;
+import lotto.view.ResultView;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class LottoTVShow {
-	public static int purchaseAmount;
+	private Lotto lotto = new Lotto();
+	private LottoResult lottoResult = new LottoResult();
 
-	public static int lottoTryCount;
+	private static final int LOTTO_PRICE_PER_SHEET = 1000;
 
-	public static List<List<Integer>> autoGeneratedLottos = new ArrayList<>();
+	public void start() {
+		int lottoTryCount = printIntro();
 
-	public static List<Integer> lastWeekPrizeNumbers = new ArrayList<>();
+		List<Lotto> generatedLottos = lotto.generatePurchagedAutoLotto(lottoTryCount);
 
-	static Lotto lotto = new Lotto();
+		List<LottoMatchResult> lottoMatchResultList = processLotto(generatedLottos);
 
-	public int calculateLottoCount(int purchaseAmount) {
-		return purchaseAmount / Lotto.LOTTO_PRICE_PER_SHEET;
+		this.lottoResult.arrangePrize(lottoMatchResultList, LOTTO_PRICE_PER_SHEET);
+		ResultView.printResult(lottoMatchResultList, lottoResult);
+	}
+
+	private int printIntro() {
+		int purchaseAmount = InputView.printInputNoticeForPurchaseAmount();
+		return calculateLottoCount(purchaseAmount);
+	}
+
+	private List<LottoMatchResult> processLotto(List<Lotto> generatedLottos) {
+		Lotto lastWeekPrizeNumbers = InputView.printLastPrizeInput();
+		int bonusNumber = InputView.printBonusNumberInput();
+		return this.compareLastWeekPrizeNumberWithAutoGeneratedLottoNumber(lastWeekPrizeNumbers, generatedLottos, bonusNumber);
 	}
 
-	static List<Integer> compareLastWeekPrizeNumberWithAutoGeneratedLottoNumber() {
-		List<Integer> lottoResultList = new ArrayList<>();
-		for (List<Integer> autoGeneratedLotto : LottoTVShow.autoGeneratedLottos) {
-			int matchCount = lotto.getMatchCount(LottoTVShow.lastWeekPrizeNumbers, autoGeneratedLotto);
-			lottoResultList.add(matchCount);
+	public List<LottoMatchResult> compareLastWeekPrizeNumberWithAutoGeneratedLottoNumber(Lotto lastWeekPrizeNumbers, List<Lotto> buyLottos, int bonusNumber) {
+		List<LottoMatchResult> lottoResultList = new ArrayList<>();
+		for (Lotto lotto : buyLottos) {
+			boolean isMatchBonus = LottoPrizePolicy.isMatchBonusNumber(lotto, bonusNumber);
+			int matchCount = lotto.getMatchCount(lastWeekPrizeNumbers, lotto);
```

Comment:
```suggestion
			int matchCount = lastWeekPrizeNumbers.countMatchNumber(lotto);
```

lotto 객체의 메서드를 호출하면서 lotto 객체 자기자신을 파라미터로 넘겨줄 필요가 있을까요?